pkg/metrics/tendermint.go

```diff
@@ -290,13 +291,15 @@ func FetchBlock(ctx context.Context, c *TendermintClient, height int64) (*Tender
 			return nil, errors.Wrap(err, "cannot unmarshal transaction")
 		}
 		block.Transactions = append(block.Transactions, &tx)
+		block.TransactionHashes = append(block.TransactionHashes, sha256.Sum256(rawTx))
```

Comment:
q: why not do the hashing in the sync range https://github.com/iov-one/block-metrics/pull/17/files#diff-968b1365c6d7fe1713a5835749794fc5R106 instead? Then you don't need to store the hashes in TendermintBlock

Reply:
It feels like a much more fluent api. I would expect tendermint to return these, but they don't because of performance issues probably. 

Yet, they do return a transaction by hash.

It does not make sense to save memory as these structs are being thrown away by the gc as they are being processed. We also deserialise the transaction body there, so I figured that grouping these calculations and allowing `Sync` method to just use precalculated data is good. Also because `Sync` is quite big already and `FetchBlock` is quite straightforward.